.github/workflows/azure-e2e-test.yml

```diff
@@ -31,6 +31,17 @@ jobs:
         echo "Generated unique version for the image as: ${unique_version}"
         echo "image-version=${unique_version}" >> "$GITHUB_OUTPUT"
 
+  read-properties:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Read properties from versions.yaml
+      run: |
+        go_version="$(yq '.tools.golang' versions.yaml)"
+        [ -n "$go_version" ]
```

Comment:
if the golang key doesn't exist yq will return `null`, this check doesn't work I think

Reply:
Used above suggestion to extract version so that it fails if unset. Thanks